releasenotes/notes/make_quadratic_cqm-bbcc2957eebbfad9.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    added make_quadratic_cqm - like make quadratic, but returns a CQM treating  auxillary variable constraints as proper constraints instead of adding them to the objective
```

Comment:
``` ``make_quadratic_cqm() `` ```. Also capitalization, some extra spaces. Also not sure what the word "proper" is adding.